integration/configs/content_share_screen_capture_test.config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "`": 1,
+  "name": "content_share_screen_capture_test.config.json %ts",
+  "grids": [
+  ],
+  "tests": [
+    {
+      "name": "Content Share Screen Capture %ts",
+      "description": "Screen sharing using content share",
+      "tupleSize": 1,
+      "noOfThreads": 1,
+      "testImpl": "ContentShareScreenCapture.js",
+      "payload": {
+        "url": "https://xijghn8zxf.execute-api.us-east-1.amazonaws.com/Prod/v2",
```

Comment:
Shouldn't the url be set to "http://localhost:8080/" like in other config.json files?